go/types/encode_human_readable.go

```diff
@@ -8,20 +8,25 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"math"
 	"strconv"
+	"time"
 
 	"github.com/attic-labs/noms/go/d"
 	"github.com/attic-labs/noms/go/util/writers"
 	humanize "github.com/dustin/go-humanize"
 )
 
+const ReadableEncodingDateFormat = "2006-01-02T15:04:05-0700"
```

Comment:
The one we had was ISO 8601, I'm not sure what that conforms to in the golang definitions or why it's not there but I made the switch to RFC3339.